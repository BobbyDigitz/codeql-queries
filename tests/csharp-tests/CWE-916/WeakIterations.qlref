CWE-916/WeakIterations.ql