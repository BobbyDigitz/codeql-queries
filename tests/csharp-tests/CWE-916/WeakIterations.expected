edges
| Test.cs:24:27:24:30 | 1000 : Int32 | Test.cs:25:66:25:76 | access to local variable interations |
nodes
| Test.cs:21:66:21:69 | 1000 | semmle.label | 1000 |
| Test.cs:24:27:24:30 | 1000 : Int32 | semmle.label | 1000 : Int32 |
| Test.cs:25:66:25:76 | access to local variable interations | semmle.label | access to local variable interations |
subpaths
#select
| Test.cs:21:66:21:69 | 1000 | Test.cs:21:66:21:69 | 1000 | Test.cs:21:66:21:69 | 1000 | Use of $@. | Test.cs:21:66:21:69 | 1000 | hardcoded weak iterations |
| Test.cs:25:66:25:76 | access to local variable interations | Test.cs:24:27:24:30 | 1000 : Int32 | Test.cs:25:66:25:76 | access to local variable interations | Use of $@. | Test.cs:24:27:24:30 | 1000 | hardcoded weak iterations |
